corehq/apps/dump_reload/sql/dump.py

```diff
@@ -167,6 +171,7 @@
     # RepeatRecord, but this does not seem like a good idea.
     FilteredModelIteratorBuilder('repeaters.SQLRepeatRecord', SimpleFilter('domain')),
     FilteredModelIteratorBuilder('repeaters.SQLRepeatRecordAttempt', SimpleFilter('repeat_record__domain')),
+    FilteredModelIteratorBuilder('saved_reports.ScheduledReportLog', SimpleFilter('domain')),
```

Comment:
@mjriley Can you confirm that this is correct - if we moved a domain between servers, we would want to include ScheduledReportLog info in the dump?

Reply:
ping

Reply:
This is a low-priority model, but yes, for completeness, I'd think this would be included in a domain migration. This just tracks, for auditability purposes, whether a report was sent. If that history needs to be maintained on the new server, then this information should be moved.